test/connectivity/NetworkController/IOfflineController.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    interface IOfflineController
```

Comment:
Suggest to rename as "INetworkController" or "INetworkSwitcher"

Reply:
why do we need to have 2 different implementation (disable/enable network interface and accept/drop all packets) for offline scenario? do we expect edgeHub will have different behavior?

Reply:
+1
Also consider having one method SetStatus with parameters Offline, Ethernet/Online, Satellite, Cellular, etc.